docker-compose.yml

```diff
@@ -24,23 +26,17 @@ services:
       - REFORM_SERVICE_NAME
       - REFORM_TEAM
       - REFORM_ENVIRONMENT
-    volumes:
-      - ./build/install/job-scheduler:/opt/app/
     ports:
-      - 8484:8484
+      - $SERVER_PORT:$SERVER_PORT
     depends_on:
-      job-scheduler-database:
+      database:
         condition: service_healthy
-  job-scheduler-database:
+  database:
     build:
       context: docker/database
-    image: docker.artifactory.reform.hmcts.net/reform/job-scheduler-database
+    image: docker.artifactory.reform.hmcts.net/job-scheduler/db
```

Comment:
is this change necessary? not in line with current convention in other repos as far as I can tell

Reply:
not necessary but considered as a better approach - it fits to box now 😆 . more adequate reasoning:
- `reform` contains in the subdomain, the following path is redundant
- parent is service definition, children - service component
- everybody knows db is database

conclusion: whole path is readable, quicker readable than before, `docker images` will be easier to understand too because of less characters to print out

could be up for vote unless it is defined somewhere as a documentation how you should define docker images then this change has to be reverted

Reply:
If you plan to update all the other existing services (and everyone agrees) then go for it.
It would be quite weird to have all the images in `/reform` and job-scheduler separately :)

Reply:
if that's the case, best to stick with previous approach